### DISCUSSION TITLE
Including Repology link in Installation docs

### DISCUSSION THREAD

**Discussion by chrisdebian** [vlang/v#20536] [Ideas]
Hi, all.

I just found out about Vlang today, and was just reading the Installation docs. When I got to the part about distro-specific packages, I thought I'd have a quick look at Repology;

https://repology.org/project/vlang/badges

It was good to see some distros running a recent build. I was wondering whether anyone is thinking about adding a link to Repology, to the install docs? This would help new users understand which distros have Vlang available, and perhaps would motivate others to create a build for another distro, that doesn't already have one.

Related to this, I wondered whether consideration has been given to creating a Flatpak build?

https://flatpak.org/

https://docs.flatpak.org/en/latest/

With thanks,

Chris (chris_debian)

**Comment by JalonSolov** [comment]
Well... V doesn't have any official packages, yet, other than the zip files on github.  Even with those, the preferred method of installing V is to clone the repo and build it.  See https://github.com/vlang/v#installing-v-from-source

**Reply by chrisdebian** [reply]
Ok, thank you.
